app/routes/notices.tsx

```diff
@@ -0,0 +1,412 @@
+      </div>
+      <CardGroup>
+        <NoticeCard
+          name="LIGO/Virgo/KAGRA"
+          href="https://gcn.gsfc.nasa.gov/lvc_events.html"
+          tags={['gw']}
+          selectedTags={tagNames}
+        >
+          Gravitational-wave transients detected by the LIGO, Virgo, and KAGRA
+          network.
+        </NoticeCard>
+        <NoticeCard
+          name="IceCube"
+          href="https://gcn.gsfc.nasa.gov/amon_icecube_gold_bronze_events.html"
+          tags={['nu']}
+          selectedTags={tagNames}
+        >
+          High-energy astrophysical neutrino event candidates detected by
+          IceCube, aggregated by AMON.
+        </NoticeCard>
+        <NoticeCard
+          name="HAWC"
+          href="https://gcn.gsfc.nasa.gov/amon_hawc_events.html"
+          tags={['gamma']}
+          selectedTags={tagNames}
+        >
+          High-energy gamma rays detected by HAWC, aggregated by AMON.
+        </NoticeCard>
+        <NoticeCard
+          name="IceCube–HAWC Coincidences"
+          href="https://gcn.gsfc.nasa.gov/amon_nu_em_coinc_events.html"
+          tags={['gamma', 'nu']}
+          selectedTags={tagNames}
+        >
+          Coincidences between IceCube neutrino and HAWC gamma-ray events,
+          aggregated by AMON.
+        </NoticeCard>
+        <NoticeCard
+          name="IceCube Cascades"
+          href="https://gcn.gsfc.nasa.gov/amon_icecube_cascade_events.html"
+          tags={['nu']}
+          selectedTags={tagNames}
+        >
+          High-energy cascades detected by IceCube, aggregated by AMON.
+        </NoticeCard>
+        <NoticeCard
+          name="CALET"
+          href="https://gcn.gsfc.nasa.gov/calet_triggers.html"
+          tags={['gamma']}
+          selectedTags={tagNames}
+        >
+          Gamma-ray transients detected by the GBM instrument on CALET.
+        </NoticeCard>
+        <NoticeCard
+          name="MAXI"
+          href="https://gcn.gsfc.nasa.gov/maxi_grbs.html"
+          tags={['x-ray']}
+          selectedTags={tagNames}
+        >
+          X-ray transients detected by the MAXI instrument on the ISS.
+        </NoticeCard>
+        <NoticeCard
+          name="Fermi GBM"
+          href="https://gcn.gsfc.nasa.gov/fermi_grbs.html"
+          tags={['gamma']}
+          selectedTags={tagNames}
+        >
+          GRBs detected by the GBM instrument on Fermi.
+        </NoticeCard>
+        <NoticeCard
+          name="Fermi LAT"
+          href="https://gcn.gsfc.nasa.gov/fermi_lat_mon_trans.html"
+          tags={['gamma']}
+          selectedTags={tagNames}
+        >
+          GRBs and gamma-ray transients detected by the LAT instrument on Fermi.
+        </NoticeCard>
+        <NoticeCard
+          name="Swift GRBs"
+          href="https://gcn.gsfc.nasa.gov/swift_grbs.html"
+          tags={['gamma', 'x-ray', 'uv', 'optical']}
+          selectedTags={tagNames}
+        >
+          GRBs detected by the Neil Gehrels Swift Observatory.
```

Comment:
You spell out the full name of Swift here, but not the full name of other observatories.  We should be consistent, either way.